content/blog/esp32-p4-eye-is-here/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "ESP32-P4 is here!"
```

Comment:
I suggest to change the title to: `Introducing ESP32-P4-EYE: A Powerful Vision Development Board for Edge AI`